components/menu/stories/menu.stories.js

```diff
@@ -114,12 +121,24 @@ Default.args = {
 					isActive: true,
```

Comment:
![image](https://github.com/user-attachments/assets/b9513875-8835-4fcf-92d9-da290e6c1512)

Realizing now with the down state transform that this focused menu item is also active (and has been this whole time). To me it feels a little funny to have both but will leave it up to your discretion whether the `isActive: true` here should stay or go. 🙂

Reply:
added to the fast-follow ticket